pkg/operation/botanist/component/istio/charts/istio/istio-istiod/templates/namespace.yaml

```diff
@@ -6,4 +6,4 @@ metadata:
   labels:
     istio-operator-managed: Reconcile
     istio-injection: disabled
-{{- end }}
+{{ end -}}
```

Comment:
Any particular reason for this change? I think that it does not really make a difference here, but we usually use the form that was there before.
WDYT?

Reply:
With the latter, the whitespace is really removed which didn't work with `{{- end }}`.

Reply:
Good enough for me.